contrib/inventory/collins.py

```diff
@@ -169,8 +169,9 @@ def run(self):
         print(data_to_print)
         return successful
 
-    def find_assets(self, attributes={}, operation='AND'):
+    def find_assets(self, attributes=None, operation='AND'):
         """ Obtains Collins assets matching the provided attributes. """
+        attributes = attributes or {}
```

Comment:
For these, it is best to check against None in case the function could validly be passed an empty container.

Reply:
Thanks. I used a regex to rewrite the patch and pushed an updated commit.